### ISSUE TITLE
Dimensionless mode

### ISSUE THREAD

**Issue by gideonsimpson** [libAtoms/JuLIP.jl#74]
This module should really have the analog of ``units lj`` that LAMMPS offers for making computations in dimensionless units.

**Comment by cortner** [comment]
I agree - its been on my TODO list for a while

**Comment by cortner** [comment]
@jameskermode does ASE offer this?

**Comment by jameskermode** [comment]
No, ASE requires all atoms to have a chemical species and is tied to A/eV/amu unit system.

The specific problem Gideon encountered was the requirement for atoms to have a chemical species assigned, which doesn't make sense for a model system. This came up because `minimise!` tries to construct a preconditioner by inferring nearest neighbour distance from chemical species.

**Comment by cortner** [comment]
Maybe one could reserve a chemical symbol (e.g. "No" for "no unit") and just bypass ASE for that case. Not very elegant though. Probably better to wait until ASE is formally decoupled and then introduce a chemical symbol for dimensionless model systems.

**Comment by cortner** [comment]
@gideonsimpson The branch `jatoms`  is a refactoring of `JuLIP` that allows "Made-up" species. If you specify for me what you'd like to see, then I'll make sure it works ok. NB once this is merged you can use JuLIP without ASE, which will make it much easier to install, use as well as much faster.

**Comment by jameskermode** [comment]
We will have to think about how to define a consistent label to chemical species mapping when converting to ASEAtoms

**Comment by cortner** [comment]
I would say, just don't convert anything that isn't a "real" species.

**Comment by cortner** [comment]
does `ASE` allow "X"?

**Comment by gabor1** [comment]
In a way, that's correct. afterall, what is ASE going to do with unidentified atoms? its potentials look at the species to decide what to do. 

-- Gábor

**Comment by cortner** [comment]
this should be all settled in the `jatoms` branch. if there is a problem with conversion we can start a new issue in ASE.jl

**Comment by jameskermode** [comment]
Sorry to reopen, but hadn't answered prevsious question: ASE does allow "X", which maps to atomic number 0.

**Comment by cortner** [comment]
So that’s exactly what I’m doing as well

**Comment by cortner** [comment]
And is this intended tobe used tat way?

**Comment by jameskermode** [comment]
Yes - they suggest using `tags` array to differentiate between different types of special atoms

**Comment by cortner** [comment]
So should anything changed with respect to this?

**Comment by jameskermode** [comment]
No, if "X" <--> 0 already then this can be closed again
